### ISSUE TITLE
Snap package not loading on Manjaro Linux

### ISSUE THREAD

**Issue by bpringe** [microsoft/vscode#113637]
- VSCode Version: Not exactly sure because I can't open it or run `code`, but release in Add/Remove Software is ea3859d4 (edit: seeing the commit message it looks like 1.52.1 is the version)
- OS Version: Manjaro Linux
  - Release 20.2

Steps to Reproduce:

1. Download Visual Studio Code Snap
2. Try to open a project via the icon, nothing happens
3. Try to open a project via `code <project-dir>`, see `cannot query current apparmor profile: Invalid argument`

Tested with insiders build 4a875e23 and it also does not work. (`code` command isn't found / added to path with install, but clicking the icon does nothing).

**Comment by manuel5975p** [comment]
Just happened to me as well

**Comment by yamafaktory** [comment]
Hi there,

I'm also using Manjaro.

It seems to be directly related to the apparmor profiles for code & url-handler:

```sh
❯ aa-enabled
Yes

❯ sudo aa-status
[...]
2 profiles are in complain mode.
   snap.code.code
   snap.code.url-handler
```

**Comment by yamafaktory** [comment]
@bpringe I found the solution on the Manjaro forum https://forum.manjaro.org/t/snaps-stopped-working-after-2020-12-30-update/45518/11.

--------------------------------

TL;DR :arrow_right: bump up the kernel to `5.10` and the issue is solved :tada: :rocket: .

**Comment by bpringe** [comment]
So, I tried this, though it was my first time trying to change the kernel, and it caused my OS to not load the UI on boot. So I had to access the OS from a bootable USB + terminal to undo what I did so I could access my OS again. :smile:

Just a word of caution to anyone who hasn't done this before ^. I probably did something wrong. Anyway, instead of making the snap work, I built the AUR package instead, and this has been working great for me.
